hometask8/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "",
+  "version": "1.0.0",
+  "description": "This is a user API",
+  "main": "index.js",
+  "scripts": {
+    "hometask8": "nodemon index.js"
+  },
+  "keywords": [
+    "swagger"
+  ],
+  "license": "MIT",
+  "private": true,
+  "dependencies": {
+    "connect": "^3.2.0",
+    "swagger-tools": "0.8.*"
```

Comment:
Consider checking the following article for more convenient npm packages:
https://blog.cloudboost.io/adding-swagger-to-existing-node-js-project-92a6624b855b